tokio/src/io/mod.rs

```diff
@@ -207,11 +207,18 @@ pub use std::io::{Error, ErrorKind, Result, SeekFrom};
 cfg_io_driver! {
     pub(crate) mod driver;
 
+    mod registration;
+
     mod poll_evented;
+
     #[cfg(not(loom))]
     pub(crate) use poll_evented::PollEvented;
+}
 
-    mod registration;
+cfg_net_unix! {
```

Comment:
If we are bikeshedding...

I would consider putting this in a `tokio::io::unix` module :).